docs/quickstarts/cosign-akv-azure.md

```diff
@@ -0,0 +1,441 @@
+---
+title: Ensure that AKS clusters only run images signed with Cosign signatures and keys in AKV
+---
+
+# Ensure that AKS clusters only run images signed with Cosign signatures and keys in AKV
+
+This article provides an e2e walkthrough of the process to ensure that Azure Kubernetes Service (AKS) clusters run images that are signed with Cosign signatures and keys stored in Azure Key Vault (AKV).
+
+In this article:
+
+* Sign an image in Azure Container Registry (ACR) with Cosign signatures and keys in AKV
+* Set up an AKS cluster without Azure policy enabled
+* Install OPA Gatekeeper and Ratify
+* Set up Ratify for validating Cosign signatures using keys in AKV
+* Run signed and un-signed images to confirm only signed images are allowed for deployment
+
+> For AKS clusters with Azure policy enabled, Image integrity policy is recommended for ensuring only signed images are deployed
+
+### Prerequisite
+
+The instructions provided in this article are tailored for Linux-based operating systems.
+
+* Install [Cosign](https://docs.sigstore.dev/system_config/installation/)
+* Install and configure the latest [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+* Create or use an [ACR](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-get-started-azure-cli) for storing container images and signatures
+* Create or use an [AKV](https://learn.microsoft.com/en-us/azure/key-vault/general/quick-create-cli) for managing keys
+* Create or using an existing AKS cluster
+ 
+## Sign an image in ACR with Cosign signatures and keys in AKV
+
+1. Configure environment variables.
+
+    ```bash
+    export SUB_ID=<your subscription id>
+
+    export AKV_NAME=<your AKV name>
+    export KEY_NAME=<your key name>
```

Comment:
when I read this, i wasn't sure if I needed to provide a existing key. suggestion for additonal comment "Keys to be generated"